colabs/Similarity_tests.ipynb

```diff
@@ -0,0 +1,1113 @@
+              "                <dd><code>Colab</code></dd>\n",
+              "            </dl>\n",
+              "        </div>\n",
+              "        \n",
+              "            </div>\n",
+              "        "
+            ]
+          },
+          "metadata": {},
+          "execution_count": 6
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%time\n",
+        "train_data = pd.read_parquet(TRAIN_DATA, columns=[\"nconst\", \"tconst\", \"category\", \"primaryName\", \"knownForTitles\",\n",
+        "                                                  \"no_nominations_oscars_norm\", \"no_nominations_globes_norm\",\n",
+        "                                                  \"no_nominations_emmy_norm\", \"no_films_norm\",\n",
+        "                                                  \"average_films_rating_norm\", \"genres_code\", \"types_code\"])\n",
+        "validate_data = pd.read_parquet(VALIDATE_DATA, columns=[\"nconst\", \"tconst\", \"category\", \"primaryName\", \"knownForTitles\",\n",
+        "                                                  \"no_nominations_oscars_norm\", \"no_nominations_globes_norm\",\n",
+        "                                                  \"no_nominations_emmy_norm\", \"no_films_norm\",\n",
+        "                                                  \"average_films_rating_norm\", \"genres_code\", \"types_code\"])"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "AMzXW0oIGAqT",
+        "outputId": "e3c12643-f3e5-4c3d-e754-5a677c31b2d3"
+      },
+      "execution_count": 7,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "CPU times: user 33.3 s, sys: 7.24 s, total: 40.5 s\n",
+            "Wall time: 53.3 s\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Porównywanie dwóch aktorów"
+      ],
+      "metadata": {
+        "id": "r8q5uNEDV57T"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Wyciągnięcie pierwszego aktora"
+      ],
+      "metadata": {
+        "id": "Oum5fGAmGpzt"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Weź id aktora ze strony imdb. Wpisujemy imię i nazwisko aktora w wyszukiwarkę na imdb i kopiujemy jego id z linku, np. dla Natalie Portman id bierzemy z linku: https://www.imdb.com/name/nm0000204/?ref_=nv_sr_srsg_0"
+      ],
+      "metadata": {
+        "id": "qfBx7BfWcUdR"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "actor1_id = \"nm0000204\""
+      ],
+      "metadata": {
+        "id": "4E_wuUsfIfVM"
+      },
+      "execution_count": 8,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "if (train_data['nconst'] == actor1_id).any():\n",
```

Comment:
te ify można również zastąpić przez `if actor1_id in list(train_data['nconst']):`
nie wiem czy to będzie lepsze, ale jest to luźny pomysł

Reply:
zamieniłam